tests/jenkins/jobs/InputManifest_Jenkinsfile.txt

```diff
@@ -0,0 +1,10 @@
+   InputManifest_Jenkinsfile.run()
+      InputManifest_Jenkinsfile.library(jenkins)
+      InputManifest_Jenkinsfile.pipeline(groovy.lang.Closure)
+         InputManifest_Jenkinsfile.echo(Executing on agent [label:none])
+         InputManifest_Jenkinsfile.stage(Input Manifest Properties, groovy.lang.Closure)
+            InputManifest_Jenkinsfile.script(groovy.lang.Closure)
+               InputManifest_Jenkinsfile.readYaml({file=manifests/1.2.0/opensearch-1.2.0.yml})
+               InputManifest.asBoolean()
+               InputManifest_Jenkinsfile.echo(opensearchstaging/ci-runner:centos7)
+               InputManifest_Jenkinsfile.echo(-e JAVA_HOME=/usr/lib/jvm/adoptopenjdk-11-hotspot)
```

Comment:
Is this using the real manifest? because it looks like it should be returning `adoptopenjdk-14-hotspot` 
https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.2.0/opensearch-1.2.0.yml#L6

Reply:
No, it's a stub in `tests/jenkins/TestInputManifest.groovy`.
![Screen Shot 2021-11-15 at 12 06 46 PM](https://user-images.githubusercontent.com/542335/141823905-f861220e-12b8-43eb-bc96-12b1da5f0504.png)